src/main/java/io/excaliburfrc/robot/Constants.java

```diff
@@ -59,4 +59,8 @@
     public static final int RIGHT_LEADER_ID = 19;
     public static final int RIGHT_FOLLOWER_ID = 15;
   }
+
+  public static class LedsConstants {
+    public static final int LEDS_PORT = 15;
```

Comment:
this will fail instantly. use a PWM port in the valid range

Reply:
first of all this is temporary until we have actual ports seconds of all ok